core/services/keystore/keys/ocr2key/aptos_keyring.go

```diff
@@ -0,0 +1,113 @@
+package ocr2key
+
+import (
+	"crypto/ed25519"
+	"encoding/binary"
+	"io"
+
+	"github.com/hdevalence/ed25519consensus"
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/chains/evmutil"
+	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+	"golang.org/x/crypto/blake2s"
```

Comment:
nit/ could group libocr in its own block below these third party packages.

Reply:
thanks for the nit!